backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -302,13 +331,54 @@
                 if (props != null) {
                     Endpoint endpoint = endpointsMap.get(props.getId());
                     endpoint.setProperties(props);
+                    // Todo: NOTIF-429 backward compatibility change - Remove soon.
+                    if (typedEndpointClass.equals(CamelProperties.class)) {
+                        endpoint.getProperties(CamelProperties.class).setSubType(endpoint.getSubType());
+                    }
                 }
             })).replaceWith(Uni.createFrom().voidItem());
         }
 
         return Uni.createFrom().voidItem();
     }
 
+    private <T extends EndpointProperties> Uni<List<T>> find(Class<T> typedEndpointClass, Set<UUID> endpointIds, boolean useStatelessSession) {
```

Comment:
Where is that new method used?

Reply:
Aah -- no where, no idea if is more debris from the rebase or where it came from :D

Reply:
Removed